app/bundles/EmailBundle/Swiftmailer/Momentum/Metadata/MetadataProcessor.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * @copyright   2018 Mautic Contributors. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\Metadata;
+
+use Mautic\EmailBundle\Helper\MailHelper;
+use Mautic\EmailBundle\Swiftmailer\Message\MauticMessage;
+
+class MetadataProcessor
+{
+    /**
+     * @var array
+     */
+    private $metadata = [];
+
+    /**
+     * @var array
+     */
+    private $substitutionKeys = [];
+
+    /**
+     * @var array
+     */
+    private $substitutionMergeVars = [];
+
+    /**
+     * @var array
+     */
+    private $mauticTokens = [];
+
+    /**
+     * @var \Swift_Message
+     */
+    private $message;
+
+    /**
+     * MetadataProcessor constructor.
+     *
+     * @param \Swift_Message $message
+     */
+    public function __construct(\Swift_Message $message)
+    {
+        $this->message = $message;
```

Comment:
This property is only used in a constructor. Can we delete it?

Reply:
Confirmed, removed